lib/actions/index.js

```diff
@@ -241,3 +247,11 @@ export const setUserFromStorage = (user) => {
     dispatch(fetchUserData(user.id, user.token))
   }
 }
+
+export const openUserSound = (sound) => {
```

Comment:
Cool, i'm going to see if we need any edit functionality in our sound reducer anymore.